Detections/SigninLogs/DisabledAccountSigninsAcrossManyApplications.yaml

```diff
@@ -26,6 +26,7 @@ query: |
   | where TimeGenerated >= ago(timeframe)
   | where ResultType == "50057"
   | where ResultDescription =~ "User account is disabled. The account has been disabled by an administrator."
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), count(), applicationCount = dcount(AppDisplayName), applicationSet = makeset(AppDisplayName) by UserPrincipalName
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), count(), applicationCount = dcount(AppDisplayName), 
```

Comment:
same comment as above: if the IP is _already_ a column then let's ensure it's extracted as an entity. But we shouldn't change the queries to return 1 row per IP.

Reply:
Tested on more than one workspace with varied data, we should be good here.